cmake/modules/FindKOKKOS.cmake

```diff
@@ -32,9 +32,12 @@ IF(DEAL_II_TRILINOS_WITH_KOKKOS)
   # Let deal.II know that we have found Kokkos
   SET(KOKKOS_FOUND ON)
 ELSE()
-  FIND_PACKAGE(Kokkos
+  # temporarily disable ${CMAKE_SOURCE_DIR}/cmake/modules for module lookup
+  LIST(REMOVE_ITEM CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules/)
+  FIND_PACKAGE(Kokkos 3.6.0
```

Comment:
Should we bump this to 3.7? We decided on 3.6 so that the latest release of Trilinos worked but since we are not checking the version of Kokkos when using Trilinos, we can be more restrictive.

Reply:
Done.